draft-ietf-ppm-dap.md

```diff
@@ -2459,23 +2509,12 @@ designed to allow implementations to reduce operational costs in certain cases.
 In general, the Aggregators are required to keep state for tasks and all valid
 reports for as long as collection requests can be made for them. However, it is
 not necessary to store the complete reports. Each Aggregator only needs to store
-an aggregate share for each possible batch interval (for time-interval) or batch
-ID (for leader-selected), along with a flag indicating whether the aggregate
-share has been collected. This is due to the requirement that in the
+an aggregate share for each batch, along with a flag indicating whether the
+aggregate share has been collected. This is due to the requirement that in the
 time-interval case, the batch interval respect the boundaries defined by the DAP
 parameters; and that in leader-selected case, a batch is determined entirely by
 a batch ID. (See {{batch-validation}}.)
 
-However, Aggregators are also required to implement several per-report checks
-that require retaining a number of data artifacts. For example, to detect replay
-attacks, it is necessary for each Aggregator to retain the set of report IDs of
-reports that have been aggregated for the task so far. Depending on the task
-lifetime and report upload rate, this can result in high storage costs. To
-alleviate this burden, DAP allows Aggregators to drop this state as needed, so
-long as reports are dropped properly as described in {{input-share-validation}}.
-Aggregators SHOULD take steps to mitigate the risk of dropping reports (e.g., by
-evicting the oldest data first).
```

Comment:
I think the core idea here still applies: it's just that we're storing report ranges rather than IDs.

Reply:
OK -- I dropped this because we'll now be storing perhaps a few thousand ID ranges per time window, but no objection to keeping this text around.

Reply:
I'm just suggesting we update the text.